tests/cmds/test_legal_hold.py

```diff
@@ -191,6 +459,11 @@ def preservation_policy_response(mocker):
     return _create_py42_response(mocker, POLICY_RESPONSE)
 
 
+@pytest.fixture
+def devices_list_generator(mocker):
+    return [TEST_DEVICE_PAGE]
```

Comment:
I find it is better to actually use generators for mocks when they are supposed to be rather than lists.. It saved future headaches, in my experience.

Reply:
Might need to use `side_effect` instead of `return_value` in that case, not sure